test/tests/postprocessors/side_advection_flux_integral/tests

```diff
@@ -0,0 +1,30 @@
+[Tests]
+  [test]
+  issues = '#24289'
+  design = 'SideAdvectiveFluxIntegral.md'
+  requirement = 'The system shall be able to output computing the integral of the component of the flux vector for advection problems'
```

Comment:
```suggestion
  requirement = 'The system shall be able to output the integral of a component of an advective flux vector for problems'
```

Reply:
T is used in `GenericMaterialProperty<T, is_ad>`. I thought it is required. Should I remove T from template?

Reply:
You can write "Real" there.
I just dont know that we advect anisotropic quantities yet or ever will